snap/hooks/configure

```diff
@@ -41,4 +42,4 @@ ldconfig -X -f $snap_ld_so_conf -C $snap_ld_so_cache
 
 # We need to update the real ld.so.cache in place rather than moving
 # over the top of it, since it is the source of a bind mount.
-cat $snap_ld_so_cache > $real_ld_so_cache 
```

Comment:
this is strange though, a \n maybe?

Reply:
Yeah, there was no new line at the end of the file